src/components/IndexTable/IndexTable.tsx

```diff
@@ -0,0 +1,630 @@
+      <Sticky>
+        {(isSticky: boolean) => {
+          const stickyHeaderClassNames = classNames(
+            styles.StickyTableHeader,
+            isSticky && styles['StickyTableHeader-isSticky'],
+          );
+
+          const bulkActionClassNames = classNames(
+            styles.BulkActionsWrapper,
+            condensed && styles['StickyTableHeader-condensed'],
+            isSticky && styles['StickyTableHeader-isSticky'],
+          );
+
+          const bulkActionsMarkup = shouldShowBulkActions ? (
+            <div className={bulkActionClassNames} data-condensed={condensed}>
+              {loadingMarkup}
+              <BulkActions
+                smallScreen={condensed}
+                label={i18n.translate('Polaris.IndexTable.selected', {
+                  selectedItemsCount: selectedItemsCountLabel,
+                })}
+                accessibilityLabel={bulkActionsAccessibilityLabel}
+                selected={bulkSelectState}
+                selectMode={selectMode || isSmallScreenSelectable}
+                onToggleAll={handleTogglePage}
+                promotedActions={promotedBulkActions}
+                actions={bulkActions}
+                paginatedSelectAllText={paginatedSelectAllText}
+                paginatedSelectAllAction={paginatedSelectAllAction}
+                onSelectModeToggle={
+                  condensed ? handleSelectModeToggle : undefined
+                }
+              />
+            </div>
+          ) : null;
+
+          const stickyColumnHeaderClassNames = classNames(
+            styles.StickyTableColumnHeader,
+            hasMoreLeftColumns && styles['StickyTableColumnHeader-isScrolling'],
+          );
+
+          const headerMarkup = condensed ? (
+            <div className={styles.HeaderWrapper}>
+              {loadingMarkup}
+              {sort}
+              <Button
+                icon={EnableSelectionMinor}
+                onClick={toggleIsSmallScreenSelectable}
+              >
+                {i18n.translate('Polaris.IndexTable.selectButtonText')}
+              </Button>
+            </div>
+          ) : (
+            <div className={stickyHeaderClassNames}>
+              {loadingMarkup}
+              <div className={stickyColumnHeaderClassNames}>
+                {stickyColumnHeader}
+              </div>
+              <div
+                className={styles.StickyTableHeadings}
+                ref={stickyHeaderElement}
+              >
+                {stickyHeadingsMarkup}
+              </div>
+            </div>
+          );
+
+          const stickyContent = bulkActionsMarkup
+            ? bulkActionsMarkup
+            : headerMarkup;
+
+          return stickyContent;
+        }}
+      </Sticky>
+    </div>
+  );
+
+  const scrollBarContentStyles =
+    tableElement.current && tableInitialized
+      ? {
+          height: '1px',
+          width: tableElement.current.offsetWidth - SCROLL_BAR_PADDING,
```

Comment:
Is `tableElement.current.offsetWidth` supposed to change with every render when the browser is being resized?
It seems to stay the same and might be whats causing the scrollbar issue.

Reply:
@lhoffbeck Sounds like this might be what you noticed this morning

Reply:
Yeah that looks exactly like what I was seeing!

Reply:
@umar-k Should be fixed now - [here's the commit](https://github.com/Shopify/polaris-react/pull/3646/commits/b529ce37b42a949d74b306e42d9163b36fd489b5)